### PR TITLE
add create_database.sql and create_table.sql files

### LINKED ISSUE #8: Create database
**Is your feature request related to a problem? Please describe.**
Create scripts that will populate the database described in the [doc](https://github.com/NikitaKit1998/datakyt/blob/main/docs/diagram.md)

**Describe the solution you'd like**

* [x] file create_database.sql
* [x] file create_table.sql
* [x] README how to to expand PostgreSQL for Linux

### PR DESCRIPTION
**Pull request by OleksandrZhytnyk**
# What does this PR do?

Add create_database.sql and create_table.sql files

## Issue reference

Fixes #8

## Explanation

To create Database and set role admin

### CODE REVIEW

**Review by paulzcooper — APPROVED** [review]

**Review by paulzcooper — DISMISSED** [review]

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: Remove empty lines here and below.

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: fix indentation

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: fix indentations

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: why 65? we know exactly the maximum length of the possible status value

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: We have a special data type `name`, Please check if we can use it everywhere 
https://www.postgresql.org/docs/current/datatype-character.html#DATATYPE-CHARACTER-SPECIAL-TABLE
CC @paulzcooper

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: why do we need this date?

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: Remove commented code

**Review by aartiukh — CHANGES_REQUESTED** [review]

File: datakyt/db/create_table.sql
Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: okey

File: datakyt/db/create_table.sql
Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: When someone gets a job in a company, the admin gives the key to some product(let it be windows ), and if this man to fire, the admin close the license and it is date_of_return.

File: datakyt/db/create_table.sql
Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: done

File: datakyt/db/create_table.sql
Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: Okey, reduce to 10, max length

File: datakyt/db/create_table.sql
Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: I have read some sources and they recommend using a text type, so I changed all lines of char to text in the name.

File: datakyt/db/create_table.sql
Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: done

File: datakyt/db/create_table.sql
Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: done

File: datakyt/db/create_table.sql
Comment (aartiukh) [reply datakyt/db/create_table.sql]: Why does admin need this `date_of_return`? Any example why it is useful to have it?
Say admin gave the license to several people, one by one. What will be stored in this field?

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: `text`

File: datakyt/db/create_table.sql
Comment (aartiukh) [inline datakyt/db/create_table.sql]: `text` ?

Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: done

File: datakyt/db/create_table.sql
Comment (OleksandrZhytnyk) [reply datakyt/db/create_table.sql]: done

**Review by aartiukh — APPROVED** [review]

**Review by MykytaKyt — DISMISSED** [review]
seems good

**Comment by aartiukh** [comment]
@Alex-ty1 please wait for @paulzcooper 's approval

File: datakyt/db/create_database.sql
Comment (paulzcooper) [inline datakyt/db/create_database.sql:5]: I think we should add such a line 
`GRANT ALL ON SCHEMA public TO postgres;` 
to create_database script so `postgres` will be another super-user

Comment (aartiukh) [reply datakyt/db/create_database.sql:5]: @paulzcooper let's skip this for now and add once we see it's needed.

Comment (paulzcooper) [reply datakyt/db/create_database.sql:5]: @aartiukh okay

**Review by paulzcooper — DISMISSED** [review]

**Review by aartiukh — CHANGES_REQUESTED** [review]
Bash script is not implemented

**Review by aartiukh — DISMISSED** [review]

**Review by aartiukh — DISMISSED** [review]

**Review by MykytaKyt — DISMISSED** [review]

**Review by paulzcooper — DISMISSED** [review]

**Review by aartiukh — APPROVED** [review]

**Review by MykytaKyt — APPROVED** [review]

**Review by paulzcooper — APPROVED** [review]

**Pull request merged**
